operations/node_management/Change_Settings_in_the_Bond.md

```diff
@@ -0,0 +1,28 @@
+## Change Settings in the Bond
+
+iPXE is used to setup udev rules for interface names and bond members. The configuration of these are dynamic on boot until node customization runs (cloud-init) and sets up the conventional `/etc/sysconfig/network/ifcfg-bond0` and other neighboring files.
+
+The initial settings of the bond(s) can be changed directly in the LiveCD or with the Boot Script Service (BSS). When cabling is different than normal, there is flexibility for customizing bond links.
+
+
+### Prerequisites
+
+This procedure requires administrative privileges.
+
+
+### Procedure
+
+1. Customize the settings of the bond(s).
+   
+   When one bond is formed:
```

Comment:
Add references (for all of the branch PRs) to either:
- Metal-iPXE https://github.com/Cray-HPE/metal-ipxe/
- The lines of source code https://github.com/Cray-HPE/metal-ipxe/blob/main/script.ipxe#L140-L141 ; mention these are available on the PIT node at `/var/www/ncn-*/script.ipxe` and the master script is at `/var/www/boot/script.ipxe`. Admins would be able to apply this to each NCN, or the master script. If they apply to the master script, then the normal install procedure of running `export IPMI_PASSWORD=changeme && /root/bin/set-sqfs-links.sh` would apply that master script to all NCNs.